hack/jenkins/benchmark-dockerized.sh

```diff
@@ -48,10 +48,15 @@ export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
 export GO111MODULE=off
 
 # Install tools we need
-pushd "${KUBE_ROOT}/hack/tools" >/dev/null
```

Comment:
since we're moving to a subshell anyhow, pushd/popd doesn't make sense and takes more lines (also pushd/popd is more fragile)